src/server_manager/web_app/ui_components/outline-intro-step.js

```diff
@@ -205,15 +205,39 @@ Polymer({
           </div>
         </div>
 
-        <div id="gcp" class="card" on-tap="setUpGcpTapped">
+        <div id="gcp" class="card" on-tap="setUpGcpTapped" hidden\$="[[!_showNewGcpFlow(gcpAccountName)]]">
           <div class="card-header">
-            <div class="tag" hidden\$="[[_computeIsAccountConnected(gcpAccountName)]]">[[localize('setup-advanced')]]</div>
+            <div class="tag">[[localize('setup-beta')]]</div>
             <div class="email" hidden\$="[[!_computeIsAccountConnected(gcpAccountName)]]">[[gcpAccountName]]</div>
             <img src="images/gcp-logo.svg">
           </div>
           <div class="card-title">Google Cloud Platform</div>
           <div class="card-body">
-            <div class="description" hidden\$="[[_computeIsAccountConnected(gcpAccountName)]]">
+          <div class="description">
```

Comment:
This is a bit hard to read. Perhaps move the card htmls to functions?

Reply:
OK, I moved the card HTML into constants.  (Templates are static in polymer.)  This required fixing the postcss hook to handle multiple `<style>` tags in one file.